readme.md

```diff
@@ -4,7 +4,7 @@
 **Contributors:** [madmimi](https://profiles.wordpress.org/madmimi), [humanmade](https://profiles.wordpress.org/humanmade), [xwp](https://profiles.wordpress.org/xwp), [illuminea](https://profiles.wordpress.org/illuminea), [maor](https://profiles.wordpress.org/maor), [beccawitz](https://profiles.wordpress.org/beccawitz)  
 **Tags:** [email](https://wordpress.org/plugins/tags/email/), [forms](https://wordpress.org/plugins/tags/forms/), [mailing list](https://wordpress.org/plugins/tags/mailing-list/), [marketing](https://wordpress.org/plugins/tags/marketing/), [newsletter](https://wordpress.org/plugins/tags/newsletter/), [opt-in](https://wordpress.org/plugins/tags/opt-in/), [signup](https://wordpress.org/plugins/tags/signup/), [subscribe](https://wordpress.org/plugins/tags/subscribe/), [widget](https://wordpress.org/plugins/tags/widget/), [contacts](https://wordpress.org/plugins/tags/contacts/)  
```

Comment:
@EvanHerman Is this too many tags? I can't remember the rules off-hand but we should double-check.